packages/@dcl/sdk-commands/src/commands/quests/index.ts

```diff
@@ -0,0 +1,282 @@
+        -b, --no-browser                                              Do not open a new browser window
+        -t, --target         [URL]                                    Specifies the target Quests server. Defaults to https://quests.decentraland.org
+
+
+      Example:
+      - Creates a new Quest:
+        $ sdk-commands quests --create
+      - Creates a new Quest from a JSON file:
+        $ sdk-commands quests --create-from-a-json /Users/nickname/Desktop/my-great-quest.json
+      - List your Quests:
+        $ sdk-commands quests -l 0xYourAddress
+        $ sdk-commands quests --list 0xYourAddress
+      - Deactivate a Quest:
+        $ sdk-commands quests --deactivate 0001-your-quest-id-2222
+      - Activate a Quest:
+        $ sdk-commands quests --activate 0001-your-quest-id-2222
+  `)
+}
+
+export async function main(options: Options) {
+  const linkerPort = options.args['--port']
+  const openBrowser = !options.args['--no-browser']
+  const isHttps = !!options.args['--https']
+  const linkerOpts = { linkerPort, openBrowser, isHttps }
+
+  const { logger } = options.components
+
+  if (options.args['--target']) {
+    if (
+      !new RegExp(urlRegex).test(options.args['--target']) &&
+      !options.args['--target'].includes('http://localhost')
+    ) {
+      throw new CliError('The provided target is not a valid URL')
+    }
+  }
+  const baseURL = options.args['--target'] || 'https://quests.decentraland.org'
+
+  logger.info(`You're using ${baseURL} server`)
+
+  if (options.args['--create']) {
+    await executeCreateSubcommand(options.components, linkerOpts, baseURL)
+  } else if (options.args['--create-from-json'] && options.args['--create-from-json'].length) {
+    await executeCreateSubcommand(options.components, linkerOpts, baseURL, options.args[`--create-from-json`])
+  } else if (options.args['--list']?.length) {
+    await executeListSubcommand(options.components, linkerOpts, baseURL, options.args['--list'])
+  } else if (options.args['--activate']?.length) {
+    await executeActivateSubcommand(options.components, linkerOpts, baseURL, options.args['--activate'])
+  } else if (options.args['--deactivate']?.length) {
+    await executeDeactivateSubcommand(options.components, linkerOpts, baseURL, options.args['--deactivate'])
+  }
+}
+
+async function executeCreateSubcommand(
+  components: CliComponents,
+  linkerOpts: Omit<LinkerdAppOptions, 'uri'>,
+  baseURL: string,
+  path?: string
+) {
+  const { logger, fs } = components
+  let quest: CreateQuest | null = null
+  if (path) {
+    if (await fs.fileExists(path)) {
+      const createQuestJson = await fs.readFile(path, { encoding: 'utf-8' })
+      try {
+        quest = JSON.parse(createQuestJson) as CreateQuest
+      } catch (error) {
+        throw new CliError(`${path} doesn't contain a valid JSON`)
+      }
+
+      if (!validateCreateQuest(quest, { logger })) {
+        throw new CliError('You provided an invalid Quest JSON. Please check the documentation')
```

Comment:
maybe we can send the validation errors here ?

Reply:
ok, changing `logger.error` within `validateCreateQuest` for `throw new CliError`